### ISSUE TITLE
crater header in .sav file makes save incompatible with other emulators

### ISSUE THREAD

**Issue by gingerbeardman** [earwig/crater#6]
Usually they just save it raw.

ROM: Magical Puzzle Popils.gg would be 8192bytes long

(i noticed because my saved offsets were not showing the correct data when browsing the crater .sav)

**Comment by earwig** [comment]
You're right—I didn't think about emulator compatibility. In the future I was hoping to use one save file to store both cart RAM and system RAM when save states are implemented, so you would need a header to denote what is what. Maybe there could be a "compatibility mode", but, ugh.

**Comment by gingerbeardman** [comment]
How about a macOS package (folder) containing both the raw .sav, the save state, and whatever else?

**Comment by earwig** [comment]
Yeah, that's good, but not cross-platform. I can do something different on other systems, but we're opening a can of worms here: currently crater is content being a CLI-run program that doesn't take advantage of macOS app conventions. If I wanted to go all the way, I could package it up in an .app bundle with a GUI and preferences and whatnot, but it's a fair bit of work for little benefit at this time. We'll see, after I finish other outstanding features.

**Comment by gingerbeardman** [comment]
Of course, no problem at all.
